standalone/src/models/store/store/types.ts

```diff
@@ -1,22 +1,26 @@
-import type { Store, StoreDraft } from '@commercetools/platform-sdk';
-import type { TBuilder } from '@/core';
-import {
-  TLocalizedStringDraftGraphql,
-  TLocalizedStringGraphql,
-} from '@/models/commons';
+import { Store, StoreDraft } from '@commercetools/platform-sdk';
+import { TBuilder } from '@/core';
+import { TCtpStore } from '@/graphql-types';
 
-//StoreDraft
-export type TStoreDraft = StoreDraft;
-export type TStoreDraftBuilder = TBuilder<TStoreDraft>;
-export type TCreateStoreDraftBuilder = () => TStoreDraftBuilder;
-export type TStoreDraftGraphql = Omit<TStoreDraft, 'name'> & {
-  name: TLocalizedStringDraftGraphql | null;
-};
-//Store
-export type TStore = Store;
-export type TStoreBuilder = TBuilder<TStore>;
-export type TCreateStoreBuilder = () => TStoreBuilder;
-export type TStoreGraphql = TStore & {
-  __typename: 'Store';
-  nameAllLocales?: TLocalizedStringGraphql | null;
-};
+export type TStoreRest = Store;
+export type TStoreGraphql = TCtpStore;
+export type TStoreDraftRest = StoreDraft;
+export type TStoreDraftGraphql = StoreDraft;
+
+/**
+ * @deprecated Use `TStoreRest` or `TStoreGraphql` instead
+ */
+export type TStore = TStoreRest;
+
+/**
+ * @deprecated Use `TStoreDraftRest` or `TStoreDraftGraphql` instead
+ */
+export type TStoreDraft = TStoreDraftRest;
+
+export type TCreateStoreBuilder<
+  TStoreModel extends TStoreRest | TStoreGraphql,
+> = () => TBuilder<TStoreModel>;
+
+export type TCreateStoreDraftBuilder<
+  TStoreDraftModel extends TStoreDraftRest | TStoreDraftGraphql,
+> = () => TBuilder<TStoreDraftModel>;
```

Comment:
We don't need two builder types:
```suggestion
export type TCreateStoreBuilder<
  TStoreModel extends TStoreRest | TStoreGraphql | TStoreDraftRest | TStoreDraftGraphql,
> = () => TBuilder<TStoreModel>;
```

Reply:
here: [ca68711](https://github.com/commercetools/test-data/pull/836/commits/ca68711f512ce6a5bb713673ad6e1ff7c0f4bd43)